apix-docker/70/docker-compose.yml

```diff
@@ -2,7 +2,7 @@ version: '2'
 
 services:
   alfresco-core:
-    image: ${DOCKER_IMAGE}
+    image: ${DOCKER_IMAGE:-private.docker.xenit.eu/alfresco-enterprise/alfresco-repository-enterprise:7.0}
```

Comment:
These defaults can lead to silent failures when the image is not built. I prefer to keep them out.